lib/mix/tasks/tower_rollbar.install.ex

```diff
@@ -0,0 +1,203 @@
+if Code.ensure_loaded?(Igniter) && Code.ensure_loaded?(Tower.Igniter) do
+  defmodule Mix.Tasks.TowerRollbar.Install do
+    @example "mix igniter.install tower_rollbar"
+
+    @shortdoc "Installs TowerRollbar. Invoke with `mix igniter.install tower_rollbar`"
+    @moduledoc """
+    #{@shortdoc}
+
+    ## Example
+
+    ```bash
+    #{@example}
+    ```
+    """
+
+    use Igniter.Mix.Task
+
+    import Tower.Igniter
+
+    @default_runtime_config """
+    import Config
+
+    if config_env() == :prod do
+      config :tower_rollbar,
+        access_token: System.get_env("ROLLBAR_SERVER_ACCESS_TOKEN"),
+        environment: System.get_env("DEPLOYMENT_ENV", to_string(config_env()))
+    end
+    """
+
+    @prod_config_patterns [
+      """
+      if config_env() == :prod do
+        __cursor__()
+      end
+      """,
+      """
+      if :prod == config_env() do
+        __cursor__()
+      end
+      """
+    ]
+
+    @rollbar_config_code """
+    config :tower_rollbar,
+      access_token: System.get_env("ROLLBAR_SERVER_ACCESS_TOKEN"),
+      environment: System.get_env("DEPLOYMENT_ENV", to_string(config_env()))
+    """
+
+    @impl Igniter.Mix.Task
+    def info(_argv, _composing_task) do
+      %Igniter.Mix.Task.Info{
+        group: :tower,
+        adds_deps: [],
+        installs: [],
+        example: @example,
+        only: nil,
+        positional: [],
+        composes: [],
+        schema: [],
+        defaults: [],
+        aliases: [],
+        required: []
```

Comment:
Lots of these have defaults and are necessary to set.

I think only needed `group` and `example` in this case.

Reply:
https://github.com/mimiquate/tower_rollbar/pull/86/commits/9fc0e505a72d372562e110adaa32b4cf4557cb04